include/EVT/dev/LCD.hpp

```diff
@@ -88,11 +86,11 @@ class LCD {
      * Clears only a certain area on the display screen.
      *
      * @param[in] width the width in pixels of the area to clear. Range: 0-127
-     * @param[in] height the height in pixels of the area to clear. Range: 0-63
+     * @param[in] numbPages the height in pixels of the area to clear. Range: 0-63
```

Comment:
This documentation should be updated to reflect the purpose of the new variable. Also, for my sanity, can we call this `numPages`, instead of `numbPages`?